lib/generate.js

```diff
@@ -20,13 +20,14 @@ function heightAscThanNameComparator(a, b) {
  * options object with the following keys:
  *
  * @param   {Object}                [options]
- * @param   {Object[]}              [options.imgs]        Array of `{ svg: Buffer, id: String }`
- * @param   {number}                [options.pixelRatio]  Ratio of a 72dpi screen pixel to the destination pixel density
- * @param   {boolean}               [options.format]      If true, generate {@link DataLayout}; if false, generate {@link ImgLayout}
- * @param   {boolean}               [options.maxIconSize] optional, overrides the max_size in mapnik
+ * @param   {Object[]}              [options.imgs]                 Array of `{ svg: Buffer, id: String }`
+ * @param   {number}                [options.pixelRatio]           Ratio of a 72dpi screen pixel to the destination pixel density
+ * @param   {boolean}               [options.format]               If true, generate {@link DataLayout}; if false, generate {@link ImgLayout}
+ * @param   {boolean}               [options.maxIconSize]          optional, overrides the max_size in mapnik
  * @param   {boolean}               [options.removeOversizedIcons] optional, if set, filters out icons that mapnik says are too big
- * @param   {Function}              callback              Accepts two arguments, `err` and `layout` Object
- * @return  {DataLayout|ImgLayout}  layout                Generated Layout Object with sprite contents
+ * @param   {boolean}               [options.stretchMetadata]      optional, if set, includes stretch metadata (`content`, `stretchX`, `stretchY`)
```

Comment:
should we only say it is used when `format` is `true`?

Reply:
Good call!